build/source/driver/multi_driver.f90

```diff
@@ -264,8 +265,12 @@ program multi_driver
 character(len=256)               :: timeString                 ! protion of restart file name that contains the write-out time
 character(len=256)               :: restartFile                ! restart file name
 character(len=256)               :: attrFile                   ! attributes file name
+! open MP functions
+integer(i4b)                     :: omp_get_thread_num,omp_get_num_threads
```

Comment:
Are these still used or can they be removed?

Reply:
The function `omp_get_num_threads` is still used. This has been retained and the other function names have been removed.

Note that we use the code
```fortran
nThreads = 1
 !$ nThreads = omp_get_num_threads()
```

and the line `!$ nThreads = omp_get_num_threads()` is interpreted as a comment if the `openMP` libraries are not linked.